lib/v2/hit/hitgs.js

```diff
@@ -0,0 +1,64 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const host = 'http://hitgs.hit.edu.cn';
+
+    const response = await got({
+        method: 'get',
+        url: host + '/tzgg/list.htm',
+        headers: {
+            Referer: host,
+        },
+    });
```

Comment:
got request can be simplified to
```suggestion
    const response = await got(host + '/tzgg/list.htm', {
        headers: {
            Referer: host,
        },
    });
```